packages/database/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- [changed] Added internal HTTP header to the WebChannel connection.
```

Comment:
I don't think database uses WebChannel? or are you using it as a general term for realtime connection?

Reply:
WebSockets. Will fix.